src/components/list/list.js

```diff
@@ -190,20 +182,12 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
           secondaryItem = buttonWrapper[0];
         }
 
-        // Check for a secondary item and move it outside
-        if ( secondaryItem && (
-                secondaryItem.hasAttribute('ng-click') ||
-                ( tAttrs.ngClick &&
-                isProxiedElement(secondaryItem) )
-            )) {
-          // When using multiple secondary items we need to remove their secondary class to be
-          // orderd correctly in the list-item
-          if (hasSecondaryItemsWrapper) {
-            secondaryItem.classList.remove('md-secondary');
-          }
-          tEl.addClass('md-with-secondary');
-          container.append(secondaryItem);
-        }
+        // When using multiple secondary items we need to remove their secondary class to be
+        // orderd correctly in the list-item
+        secondaryItem.classList.remove('md-secondary');
```

Comment:
There is no need, because, when iterating through a jqLite array, we retrieve always plain DOM elements.
So wrapping the element for one expression, is not needed in my opinion. In addition to that, I just transferred this statement from the previous child block.